core/ledger/kvledger/txmgmt/privacyenabledstate/snapshot_record.proto

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+syntax = "proto3";
+
+option go_package = "github.com/hyperledger/fabric/core/ledger/kvledger/txmgmt/statedb/privacyenabledstate";
+
+package privacyenabledstate;
+
+message SnapshotRecord {
+    bytes key = 1;    
+    bytes value = 2;
+    bytes metadata = 3;
+    bytes version = 4;
```

Comment:
nit. 

Keeping all bytes look nice but we could use `version` as-is. Currently, we convert version to bytes during the export and then convert it back to version during the import. By using version as-is, we could reduce 8 lines of code (I don't think performance will improve) but the code would look a bit leaner. 

If we decide to use `EncodeBytes()` instead of proto, we can keep this as bytes.

Reply:
Yeah, I missed the first part about being a proto message :-) 

Okay. Agree.